src/main/java/com/ldtteam/structures/client/BlueprintHandler.java

```diff
@@ -83,16 +109,29 @@ public void draw(final Blueprint blueprint, final BlockPos pos, final MatrixStac
      */
     public void cleanCache()
     {
-        final long now = System.currentTimeMillis();
-        final Iterator<Int2LongMap.Entry> iter = evictTimeCache.int2LongEntrySet().iterator();
+        long now = System.currentTimeMillis();
```

Comment:
we can only use one "now"

Reply:
I might be "overdoing it", wondered if so much time passed after all of the above, to create a new one lower.